### PR TITLE
add OOBM handler for new chat message

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

**Comment by songgao** [comment]
@mmaxim Does this look reasonable to you?

**Comment by mmaxim** [comment]
@songgao I was thinking chat activity would be more general, and new messages would just be one of the possible things we would send through this channel. A couple others might be like: message read, other person is typing, etc. Could we frame the channel more in terms of chat activity? Let me know if that doesn't make sense!

File: go/service/gregor.go [hunk 1]
```diff
@@ -841,21 +841,21 @@ func (g *gregorHandler) newChatMessage(ctx context.Context, m gregor.OutOfBandMe
 		return err
 	}
 
-	var msg keybase1.Message
+	var activity keybase1.ChatActivity
 
-	// TODO: decrypt message from body and fill into plaintext msg
+	// TODO: fill into activity struct and decrypt message if necessary
```
Comment (mmaxim) [inline go/service/gregor.go]: Are we able to do this now after @patrickxb 's PR?

Comment (songgao) [inline go/service/gregor.go]: hmm.. This also depends on the structure of the OOBM. Do we have that already as well?

Comment (mmaxim) [inline go/service/gregor.go]: You can just assume it was have  `MessageBoxed` in the payload.

**Comment by songgao** [comment]
@mmaxim could you take another look? :)

**Comment by mmaxim** [comment]
👍

**Pull request merged**